pkgs/development/cuda-modules/cuda/overrides.nix

```diff
@@ -69,8 +69,8 @@ attrsets.filterAttrs (attr: _: (builtins.hasAttr attr prev)) {
 
   cuda_compat = prev.cuda_compat.overrideAttrs (
     prevAttrs: {
-      env.autoPatchelfIgnoreMissingDeps =
-        prevAttrs.env.autoPatchelfIgnoreMissingDeps + " libnvrm_gpu.so libnvrm_mem.so libnvdla_runtime.so";
+      autoPatchelfIgnoreMissingDeps =
+        prevAttrs.autoPatchelfIgnoreMissingDeps ++ ["libnvrm_gpu.so" "libnvrm_mem.so" "libnvdla_runtime.so"];
```

Comment:
Does nixfmt not expand this?

Reply:
I forgot to format the modified lines with `nixfmt` -- fixed in latest.